pints/__init__.py

```diff
@@ -80,3 +80,9 @@ def version(formatted=False):
 from _mcmc import MCMC
 from _mcmc._adaptive import AdaptiveCovarianceMCMC, adaptive_covariance_mcmc
 
+#
+# Basic plots
+#
+import _plot as plot
+
```

Comment:
Once you've changed it to `plot.py` you can also add documentation to `pints/docs/source`. Then copy paste another module's `rst` file (for example `error_measures.rst` --> `plot.rst`) and update it. Also make sure to add your new file to the main `index.rst` (where there's an alphabetically ordered list). You can test locally by running `make html` from `pints/docs` and then checking the stuff in `pints/docs/build` in your browser.